x/evm/keeper/msg_server.go

```diff
@@ -758,9 +776,15 @@ func (s msgServer) RetryFailedEvent(c context.Context, req *types.RetryFailedEve
 	s.Logger(ctx).Info(
 		"re-queued failed event",
 		types.AttributeKeyChain, chain.Name,
-		"eventID", req.EventID,
+		"eventID", event.GetID(),
 	)
 
+	funcs.MustNoErr(ctx.EventManager().EmitTypedEvent(&types.EVMEventConfirmed{
```

Comment:
This is a bit weird tbh

Reply:
also prefer to have separate event for this